rabbitmq.drush.inc

```diff
@@ -106,9 +105,15 @@ function drush_rabbitmq_worker($queueName) {
     $consumer->consume($queueName);
   }
   catch (InvalidWorkerException $e) {
-    return drush_set_error(dt("Worker for queue @name does not implement the worker interface.", $queueArgs));
+    return drush_set_error(dt(
+      'Worker for queue @name does not implement the worker interface.',
```

Comment:
Good idea to replace the double quotes by single quotes, but the line length is not limited outside arrays and comments, so no line wrapping is expected after the "dt("